lib/run.go

```diff
@@ -266,7 +266,7 @@ func genDeplist(acipath string, reg registry.Registry) ([]string, error) {
 }
 
 func (a *ACBuild) mirrorLocalZoneInfo() error {
-	zif, err := os.Readlink("/etc/localtime")
+	zif, err := filepath.EvalSymlinks("/etc/localtime")
```

Comment:
Is this preferable to the rkt approach for some reason? https://github.com/coreos/rkt/blob/master/stage1/init/init.go#L106-L110
if so we should probably switch there too